.github/workflows/php.yaml

```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-version: ['7.1', '7.2', '7.3', '7.4']
+        php-version: ['7.3', '7.4']
```

Comment:
tady by se mohla přidat i nějaká 8.x verze, ne?

Reply:
See #143. Done in separate PR, to keep this one focused on one thing (ie. to get it working with the new coding standard).